README.md

```diff
@@ -75,20 +75,24 @@ Once running, requests can be directed to Envoy server at `localhost:3000`.
 
 You can run a [Turbo] [ANS-104] data item bundler as a sidecar to the ar.io gateway service. This will allow the deployed system to accept data items and bundle them into a single transaction before submitting them to the network. The bundler's APIs will be reachable at the `/bundler/` path. For more information on its APIs, you can access docs at `/bundler/api-docs/`.
 
-To get started, supply the required environment variables in `.env` for the integration, most notably:
+Note: A local bundler configured to integrate with an ar.io node relies upon GraphQL indexing of recently bundled and uploaded data to manage its pipeline operations. The ar.io node should have its indexes synced up to Arweave's current block height minus 18 blocks before starting up the bundler's services stack.
+
+Bundling services are most easily managed via an independent docker compose file whose services share their network with that of the core services docker compose stack. This allows you to spin the services up when your core service is prepared to integrate with it, or down whenever you want without affecting your core services stack.
+
+To get started, supply the required environment variables in an environment variables file (e.g. `.env.bundler`) for the integration, most notably:
 
 - `BUNDLER_ARWEAVE_WALLET`: a stringified JWK wallet used for uploading bundles to Arweave.
 - `ALLOW_LISTED_ADDRESSES`: a comma-separated list of allowed uploader wallet addresses (normalized). See [Managing Bundler Access](#managing-bundler-access) for more permissioning options.
 
-See the `.env.bundler.example` file for other configuration options, including settings for serving bundler-uploaded data items instantly from your gateway.
+See the `.env.bundler.example` file for other important configuration options, including settings for serving bundler-uploaded data items instantly from your gateway.
 
-Once environment variables are set, run docker compose with the `bundler` profile.
+Once environment variables are set, run docker compose with the bundler-specific compose file.
 
 ```shell
-docker compose --profile bundler up
+docker compose --env-file ./.env.bundler --file docker-compose.bundler.yaml up upload-service
```

Comment:
It seems to work fine, but what's the reason for specifying `upload-service` here rather than just leaving service off?

Reply:
Good point. Omitted: 9d0710e